app/helpers/team_helper.rb

```diff
@@ -1,5 +1,16 @@
+#:reek:UtilityFunction
 module TeamHelper
+  WHITELISTED_DOMAINS = %w[.mil .gov .fed.us].freeze
+
   def can_edit_teams?(user)
     !user.teams.empty? || user.admin?
   end
+
+  def can_create_teams?(user)
+    WHITELISTED_DOMAINS.any? { |domain| user.email.end_with? domain } || user.admin?
```

Comment:
~~Maybe consider just returning true if admin, otherwise do the domain check? As is, the admin check gets lost at the end of a long, busy line.~~

Actually, you could move the `whitelisted_user?` method from the policy to this helper, and include this helper in the policy.